.gitmodules

```diff
@@ -112,3 +112,9 @@
 [submodule "software/example-freertos-blinky"]
 	path = software/example-freertos-blinky
 	url = https://github.com/sifive/example-freertos-blinky.git
+[submodule "software/example-freertos-minimal"]
```

Comment:
In the readme, can you add a comment indicating this example is minimal starter for their FreeRTOS application